docs/source/cpp_api/core_resources.rst

```diff
@@ -202,3 +214,22 @@ namespace *raft::resource*
      :project: RAFT
      :members:
      :content-only:
+
+User-defined resources
```

Comment:
We should call this something else. Just adding a note so it doesn't get lost with the other rename.